package.json

```diff
@@ -17,18 +17,18 @@
     "npm": ">=8"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.36.1",
-    "@typescript-eslint/parser": "^5.36.1",
-    "eslint": "^8.23.0",
-    "eslint-import-resolver-typescript": "^3.4.1",
+    "@typescript-eslint/eslint-plugin": "^5.38.1",
+    "@typescript-eslint/parser": "^5.38.1",
+    "eslint": "^8.24.0",
+    "eslint-import-resolver-typescript": "^3.5.1",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-node": "^11.1.0",
     "prettier": "^2.7.1",
-    "typescript": "^4.8.2"
+    "typescript": "^4.8.4"
   },
   "dependencies": {
-    "@typescript/vfs": "1.0.0",
-    "@types/jest": "^29.0.3",
-    "jest": "^29.0.3"
+    "@typescript/vfs": "1.4.0",
```

Comment:
We have to keep this at 1.0.0. We use this pin to avoid npm from hoisting this dependency in packages/protoplugin. It's a terrible hack.